libraries/AP_HAL_ChibiOS/hwdef/EAH743QF/README.md

```diff
@@ -0,0 +1,86 @@
+# Ewing Aerospace H743QF Flight Controller
+
+The Ewing Aerospace H743QF is a flight controller produced by [Ewing Aerospace](https://ewingaerospace.com/).
+
+## Features
+
+ - STM32H743 microcontroller
+ - ICM42688 IMU
+ - BMP390 barometer
+ - MAX7456 OSD
+ - 6 UARTs
+ - 6 PWM outputs
+
+## Pinout
+
+![Ewing Aerospace H743QF v1](Ewing_Aeropsace_H743QF_Board_Top.png "Ewing Aerospace H743QF v1")
+![Ewing Aerospace H743QF v1](Ewing_Aeropsace_H743QF_Board_Bottom.png "Ewing Aerospace H743QF v1")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
```

Comment:
should have DMA info....a table of this form:
UART Port Mapping:
Port      UART      Protocol  TX DMA  RX DMA
--------- --------- ---------- ------- -------
0         USB       MAVLink2  ✘       ✘
1         USART1    GPS       ✔       ✔
2         USART2    DisplayPort  ✔       ✔
3         USART3    SmartAudio       ✔       ✔
4         UART4     RCIN      ✘       ✘
5         UART5     None      ✘       ✘  TX5 not pinned out
6         USART6    None      ✘       ✘

Reply:
its required...since this is now the ONLY user doc in the wiki, if they want to change a port to something that might need DMA, like an overloaded osd screen or RC or high speed PPP, they would otherwise have to know how to setup a build environment, configure the board, and know how to find DMA resolver in the artfacts and know how to interpret it

Reply:
btw
Serial 6 should be defaulted to RCIN protocol, Serial 3 is SmartAudio, not analog OSD which is non-existant, gps and smartAudio do not need DMA so if there is a resolver issue you can remove dma here if it helps